libs/libucontext/Makefile

```
@@ -0,0 +1,61 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libucontext
+PKG_VERSION:=1.2
+PKG_RELEASE:=1
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).zip
```

```suggestion

PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).zip
```

i'm okay if you still prefer zip, anyway please add a line break here.

---

No I don't prefere zip :-(
It seems i am a little bit unconcentrated and also confused not missing any hints from you :-( Sorry!